frontend-app/src/amit-ui-task/index.html

```diff
@@ -0,0 +1,104 @@
+</head>
+<body>
+    <header class="header">
+        <img src="./images/logo.png" alt="medium.com" class="logo">
+        <nav>
+            <ul class="top_nav_links">
+                <li>
+                    <button class="signin pointer" id="sign-in-btn" onclick= "loginmodal()" >Sign in</button>
+                </li>
+                <li>
+                    <button class="get-started pointer" onclick= "signupModal()" >Get Started</button>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <div class="navbar">
+        <a href="https://medium.com/nulogy"> <img src="./images/nuology.png" alt="" class="nuology"></a>
+        <a href=""> ENGINEERING</a>
+        <a href=""> CULTURE</a>
+    </div>
+    <div class="centre-body-section">
+        <div class="centre-item">
+            <h1 style="font-size: 48px; font-family: Georgia, 'Times New Roman', Times, serif; font-weight:400 ;" >How to Use CSS Modules with <br> Create React App</h1>
+            <h3 style="font-size: 22px; font-family: Arial, Helvetica, sans-serif; font-weight: 400; color: #757575;">Short answer: you can’t. You gotta ⏏</h3>
+            
+            <div class="content">
+                <div class="left-content">
+                    <img src="./images/photo.jpeg" alt="" class="photo" > 
+                    <div class="into-data">
+                        <div class="aboutme">
+                            <a href="https://medium.com/@theinterned">Ned Schwartz</a> 
+                            <button class= "follow pointer">Follow</button>
+                            <img id="mail" class="pointer" src="./images/Screenshot 2022-02-06 at 11.33.49 PM.png" alt="" style="height: 30px; width: 30px;">
+                            
+                        </div>
+                        <div class="text">
+                            <a href="">Feb 6, 2017</a>
+                            <span>· 6 min read</span>
+                        </div>
+                    </div>
+                </div>
+                <div class="social">
+                    <a href=""><img src="./images/twt.png"/></a> 
+                    <a href=""><img src="./images/fb.png"/></a>
+                    <a href=""><img src="./images/linkedin.png" alt=""></a>
+                    <a href=""><img src="./images/link.png" alt=""></a>
+                    <a href=""><img src="./images/bookmark.png" alt=""></a>
+                    <a href=""><img src="./images/more.png" alt=""></a>
+                </div>
+            </div>
+            <div style="display: flex; justify-content: space-around; margin-top: 20px;">
+                <img src="./images/img.png" alt="">
+            </div>
+            <div >
+                <p style="font-size: 21px;"> <i> <b> Update January 18th, 2018:</b> CSS Module support has landed in alpha builds of Create React App v2 🎉. </i> </p>
+            </div>
+        </div>
+
+
+    <div class="modal" id = "login-modal">
```

Comment:
- login modal position should be in centre
- cross sign should be at top right corner of the modal
- cross sign thickness should be less